### DISCUSSION TITLE
ユーザー毎の過去投稿した記事のタグを一覧で並べる

### DISCUSSION THREAD

**Discussion by sauna1137** [increments/qiita-discussions#819] [要望・アイデア・不具合]
## 機能
ユーザー毎の過去投稿した記事のタグを一覧で並べ、押下するとタグが付いた記事一覧に遷移する

## 目的
自身、他ユーザーの投稿した同一のタグ、カテゴリの記事を参照するため
→ 多量の記事を書いているユーザーの場合に検索するのが困難な状態です

## 詳細
場所 : プロフィールページ
UI : 「フォロー中のタグ」のイメージです。
<img width="321" alt="スクリーンショット 2024-05-31 7 15 03" src="https://github.com/increments/qiita-discussions/assets/83006844/db2cf7d7-bc60-4769-a0d9-6430574117ef">

$ analyzeの拡張機能のようなイメージです。

いつも楽しく使わせていただいております！
ご検討いただけますと幸いです。すでに機能として存在している、既に検討中などありましたら申し訳ありません。

**Comment by tanaka-asami415** [comment]
@sauna1137 
リクエストありがとうございます！
「特定のユーザー」が投稿した「特定のタグがついた記事」を絞りたい、というご要望と理解いたしました。

[ユーザーページ](https://help.qiita.com/ja/articles/qiita-mypage) で、将来的にできるようになるといいなと思いつつ
開発工数と直近の開発優先度を鑑みると、着手は中々難しそうです🙇

現状の代替策として、検索オプション（`user:記事投稿ユーザー名` 、`tag:タグ名` ）で指定していただく方法がございます。
検索結果は、「新着順」や「いいね数順」等で並べ替える事が可能です。
詳細は下記のヘルプをご参照ください。
https://help.qiita.com/ja/articles/qiita-search-options

お手数ですが、こちらをご活用いただけますと幸いです。

**Reply by sauna1137** [reply]
@tanaka-asami415 
早速のご返信ありがとうございます！

下記方法は存じ上げませんでした。参考にさせていただきます。
> 現状の代替策として、検索オプション（user:記事投稿ユーザー名 、tag:タグ名 ）で指定していただく方法がございます。
> 検索結果は、「新着順」や「いいね数順」等で並べ替える事が可能です。

**Reply by gilly135** [reply]
@sauna1137 
リクエストありがとうございました。

他のユーザーの方からのリクエストと合わせて対応を検討した結果、ユーザーページから、閲覧中のユーザーを検索条件に指定した検索結果ページに遷移できるようにしました！

詳細はリリースノートをご確認ください。
https://qiita.com/release-notes/2024-07-24

>自身、他ユーザーの投稿した同一のタグ、カテゴリの記事を参照するため

こちらのご要望に対しては、以前 @tanaka-asami415 がご提案した通りになりますが、
検索結果ページで `tag:qiita` などタグの検索オプションを追加することで詳細な検索が可能です。

ご利用いただき、またご要望などございましたらお知らせいただけると嬉しいです 🙏
